config.py

```diff
@@ -9,3 +9,6 @@
 xml_footer = "</urlset>"
 
 crawler_user_agent = 'Sitemap crawler'
+
+number_calls = 1  # number of requests per call period
+call_period = 15  # time in seconds per number of requests
```

Comment:
The default should be no rate limiting, right?

Reply:
No limit would be in line with how the original code works, yes. I could not find a default option in the ratelimit package, you could set the default to a very high number as a workaround though

Reply:
One possibility would be having something like this in `crawler.py`:
```python
if number_calls is None or call_period is None:
    rate_limit_decorator = lambda func: func
else:
    rate_limit_decorator = limits(calls=config.number_calls, period=config.call_period)
```

Haven't dealt with `@sleep_and_retry`, but I believe should be possible to combine that into `rate_limit_decorator`.  Of course, this strategy comes at the cost of increasing the complexity of the code.